packages/rule-stylesheet-limits/src/rule.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * @fileoverview Checks if CSS exceeds known stylesheet limits.
+ */
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+import { IRule, RuleMetadata } from 'sonarwhal/dist/src/lib/types';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class StylesheetLimitsRule implements IRule {
+
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.interoperability,
+            description: `Checks if CSS exceeds known stylesheet limits.`
+        },
+        id: 'stylesheet-limits',
+        schema: [{
+            additionalProperties: false,
+            properties: {
+                maxImports: { type: 'number' },
+                maxRules: { type: 'number' },
+                maxSheets: { type: 'number' }
```

Comment:
The schema should specify that the number is positive and it's an integer/whole number,

Reply:
Maybe use something such as:

```js
{
    additionalProperties: false,
    definitions: {
        number: { 
            minimum: 0,
            type: 'integer'
        }
    },    
    properties: {
        maxImports: { "$ref": "#/definitions/number" },                 
        maxRules: { "$ref": "#/definitions/number" },                 
        maxSheets: { "$ref": "#/definitions/number" }               
    }
    type: ['object', 'null']
}
```

Reply:
Sounds good. Thanks for the example!